src/components/dropdown-filter-ajax/dropdown-filter-ajax.stories.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import {
+  text,
+  select,
+  boolean,
+  number
+} from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { State, Store } from '@sambego/storybook-state';
+import ImmutableHelper from '../../utils/helpers/immutable';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import notes from './notes.md';
+import DropdownFilterAjax from './dropdown-filter-ajax';
+
+const store = new Store({
+  value: ''
+});
+
+const create = (evt, component) => component.state.filter;
+
+// Shared Props
+const onChange = (evt) => {
+  store.set({ value: evt.target.value });
+  action('change')();
+};
+const autoFocus = boolean('autoFocus', false);
+const cacheVisibleValue = boolean('cacheVisibleValue', true);
+const disabled = boolean('disabled', false);
+const name = text('name', 'Name');
+const readOnly = boolean('readOnly', false);
+const timeToDisappear = number('timeToDisappear', 0);
+const label = text('label', 'Dropdown Label');
+const labelInline = boolean('labelInline', true);
+const labelWidth = labelInline ? text('labelWidth', '') : null;
+const labelAlign = labelInline ? select('labelAlign', OptionsHelper.alignBinary) : null;
+const labelHelp = text('labelHelp', 'This is help text');
+const inputWidth = text('inputWidth', '');
+const fieldHelp = text('fieldHelp', 'This is field help text');
+const fieldHelpInline = boolean('fieldHelpInline', false);
+const createText = text('createText', '');
+const createIconType = text('createIconType', '');
+const suggest = boolean('suggest', false);
+const freetext = boolean('freetext', false);
+const path = text('path', '/');
+const acceptHeader = text('acceptHeader', 'application/json');
+const rowsPerRequest = text('rowsPerRequest', '25');
```

Comment:
Could you provide it form default props and check other default props?

Reply:
👍